tests/v2/test_problem.py

```diff
@@ -170,3 +171,22 @@ def test_modify_problem():
         }
     ).set_index([PETAB_ENTITY_ID])
     assert_frame_equal(problem.mapping_df, exp_mapping_df, check_dtype=False)
+
+
+def test_sample_startpoint():
```

Comment:
Unless other asserts are added, like ensuring the samples appear to come from the expected distribution
```suggestion
def test_sample_startpoint_shape():
```